frontend/app/components/description-list-item.tsx

```diff
@@ -0,0 +1,14 @@
+import type { PropsWithChildren, ReactNode } from 'react';
+
+export interface DescriptionListItemProps extends PropsWithChildren {
+  term: ReactNode;
+}
+
+export function DescriptionListItem({ children, term }: DescriptionListItemProps) {
+  return (
+    <div className="sm:grid sm:grid-cols-3 sm:gap-4 sm:py-4">
+      <dt className="font-semibold">{term}</dt>
+      <dd className="mt-3 space-y-3 sm:col-span-2 sm:mt-0">{children}</dd>
+    </div>
```

Comment:
Here's the [style](https://play.tailwindcss.com/PlktFSCLMc?size=778x720) based on this [Description Lists](https://tailwindui.com/components/application-ui/data-display/description-lists) example.

```suggestion
    <div className="py-6 sm:grid sm:grid-cols-3 sm:gap-4">
      <dt className="font-semibold">{term}</dt>
      <dd className="mt-1 sm:col-span-2 sm:mt-0">{children}</dd>
    </div>
```